packages/playwright-core/src/server/socksClientCertificatesInterceptor.ts

```diff
@@ -297,3 +298,17 @@ export function clientCertificatesToTLSOptions(
 function rewriteToLocalhostIfNeeded(host: string): string {
   return host === 'local.playwright' ? 'localhost' : host;
 }
+
+export function rewriteOpenSSLErrorIfNeeded(error: Error): Error {
+  if (error.message !== 'unsupported')
```

Comment:
note: Unfortunately they don't set the `.code` property. Maybe we should look at the stack to make sure its an OpenSSL error?

Reply:
I think it's fine as it is.